translations/en/en.tsx

```diff
@@ -0,0 +1,13 @@
+import common from './_common.json';
+import layoutRegion from './_layoutRegion.json';
+import pageIndex from './_pageIndex.json';
+import pageRegion from './_pageRegion.json';
+
+const en = {
```

Comment:
If I decide to continue with namespaces (as opposed to one global file) I will likely automate generating this object for every supported language, such that this file will not need to be copy pasta'd for every lang.

Reply:
Could we instead start with the global file? That might be the path of least resistance, we will likely have a lot of pages and messages in common across the pages.

Reply:
Global file here being one file per language, yeah?

Reply:
There are three possible approaches IMO. 

1. Single global file / single namespace
2. Multiple files / single namespace
3. Multiple files / multiple namespaces

I think your concern is more the namespace issue, not the file composition? Having all the translation keys in one file is going to get unruly - just speaking from experience. I'm unsure if multiple namespaces will be necessary. It will also cause us to have to initialize multiple `useTranslation` hooks if we need keys from both a common and local translation files. I was going to refactor it to a single namespace if it got weird.

I'm okay with leaving it as it is and figuring out the best way to configure it as the project grows. I like the idea of separate namespaces in theory because it keeps the template strings concise, less redundancy, and keeps the translation directory itself pretty modular.

Reply:
Sounds good!